### ISSUE TITLE
better handling for h1 unsupported protocol

### ISSUE THREAD

**Issue by jbr** [http-rs/tide#470]
In order fully resolve #458, there should be better logging and possibly a more-specific http status code when a client connects with an unsupported http version

**Comment by MPThLee** [comment]
Sorry for question on this issue but, Tide didn't supports HTTP/1.0?
I'm just trying to replacing old legacy server with tide(likely just for fun). But sadly all of requests are dropped when HTTP version is 1.0. (Clients are using legacy too.)

**Comment by jbr** [comment]
Correct, Tide does not currently support HTTP/1.0. Specifically 1.1 at the moment.

**Comment by MPThLee** [comment]
Thanks to let me know. I hope that tide will supports 1.0/2.0 too. 

~~I need to switch framework now (again) lol.~~

**Comment by jbr** [comment]
1.0 is more likely to happen before 2.0, but they've both been discussed. Sorry it's not there yet!
